src/CollapsableMixin.js

```diff
@@ -1,103 +1,133 @@
 var React = require('react');
 var TransitionEvents = require('./utils/TransitionEvents');
```

Comment:
>When children are declaratively added or removed from it (as in the example above) special lifecycle hooks are called on them

We're not adding/removing elements so I don't think we can use `ReactTransitionGroup`.

I did a diff on the core `ReactTransitionEvents` and it looks like we should be able to use that.  The note in our utils says our version is modified, but the diff says otherwise.